determiner.py

```diff
@@ -0,0 +1,115 @@
+#! /usr/bin/env python3
+
+import argparse
+import binascii
+import os
+import sys
+
+## print header first 
+## print header first
+authorSignature = '\nedeterminer.py - Published by Laura Tzigiannis and Moritz Nentwig\n'
+authorSignature += '---------------------------------------------------------------\n'
+print(authorSignature)
+
+## tests
+def check_file_extension(file): 
+    file_extension = file[file.rfind("."):]
+    if file_extension == ".aes":
+        return("AES")
+    elif file_extension == ".des":
+        return("DES")
+    elif file_extension == ".3des":
+        return("3DES")
+    elif file_extension == ".idea":
+        return("IDEA")
+    elif file_extension == ".bf":
+        return("Blowfish")
+    else:
+        return("Unrecognized file extension")
+
+def check_file_header(file):
```

Comment:
tried this test for aes, des & idea and it wasn't working

Reply:
done